client/src/hooks/helpers/useIndividualRewardAllEpochs.ts

```diff
@@ -24,7 +24,14 @@ export default function useIndividualRewardAllEpochs({
             resolve({ budget: '0' });
           });
         }
-        return apiGetIndividualRewards(epoch, address!);
+
+        try {
+          return apiGetIndividualRewards(epoch, address!);
+        } catch (error) {
```

Comment:
I suggest underscore to indicate unused variable. It's strange linter didn't pick it up.
```suggestion
        } catch (_error) {
```